end-to-end-test/shared/wdio/webdriver-manager.conf.js

```diff
@@ -159,27 +174,21 @@ var config = {
     // Test reporter for stdout.
     // The only one supported by default is 'dot'
     // see also: http://webdriver.io/guide/testrunner/reporters.html
-    reporters: ['spec', 'junit', CustomReporter],
+    reporters: ['spec', 'junit'],
     reporterOptions: {
         junit: {
-            outputDir: process.env.JUNIT_REPORT_PATH || "./",
+            outputDir: process.env.JUNIT_REPORT_PATH,
             outputFileFormat: function(opts) { // optional
                 return `results-${opts.cid}.${opts.capabilities}.xml`
             }
-        },
-        custom: {
```

Comment:
we need to keep this for our custom reporting tool

Reply:
Ooops, also changes to this file was not included after rebase. Has been updated.